app/src/main/res/drawable/access_type_bg.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
put this file in drawable-v21 if the minSdk is lower then 21. Since the ripple can crash older devices

Reply:
MinSDK is 21, but good observation.